pombola/south_africa/views/person.py

```diff
@@ -244,6 +244,9 @@ def get_attendance_stats_raw(self, data):
         minister_positions_by_year = self.get_active_minister_positions(years)
 
         for x in data:
+            if x['alternate_member']:
+                # Don't count alternate member attendances
+                continue
```

Comment:
There is some refactoring that could be done to this function, but I would rather do it as part of [As a user I would like to see total meeting attendance for this MP](https://www.pivotaltracker.com/story/show/174511091) where we want to break down the member's attendances for member, non-member and alternate member attendances.